server/routes/coverrequests.js

```diff
@@ -11,19 +11,38 @@ router.post("/pendingsupervisor", (req, res) => {
     approve = "Denied";
   }
   var requestID = req.body.requestID;
-
   pool.query(
     "UPDATE coverrequests SET supervisor_status = $1 WHERE request_id = $2",
     [approve, requestID],
     (error, result) => {
       if (error) {
         throw error;
       }
-      console.log(result.rows);
     }
   );
+  if (approve === "Approved") {
+    pool.query(
+      `UPDATE shifts SET sle_id = coverer_id
+    FROM coverrequests
+    WHERE coverrequests.shift_id = shifts.shift_id`,
+      (error, result) => {
+        if (error) {
+          throw error;
+        }
+      }
+    );
+  } else {
+    pool.query(
+      `DELETE FROM coverrequests 
```

Comment:
Wait what is this? We don't want to delete anything from coverrequests. It also looks like it's not totally done so the query won't actually work. I think we should just delete this part but correct me if Im wrong.

Reply:
It's also causing the server to crash so let's definitely delete.